src/components/interfaces/MOBILE_API.xml

```diff
@@ -1403,45 +1400,48 @@
         <element name="INVALID" />
     </enum>
     
-    <enum name="ModuleType" since="4.5">
+    <enum name="ModuleType">
         <element name="CLIMATE"/>
+        <element name="SEAT"/>
         <element name="RADIO"/>
-        <element name="SEAT" since="5.0"/>
```

Comment:
Should not rearrange order of existing modules

Reply:
@jacobkeeler  rearrange removed